samples/getting-started/azure-quantum/Getting started with Qiskit and IonQ on Azure Quantum.ipynb

```diff
@@ -0,0 +1,652 @@
+      "text/plain": [
+       "<Figure size 504x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Let's see the results!\n",
+    "# aer_sim = Aer.get_backend('aer_simulator')\n",
```

Comment:
It looks like the call to simulate on Qiskit's noisy simulator is also commented out as well as the `transpile` step? If so, it may be helpful to note that above, too.

Reply:
I ended up taking this code out.